controllers/specialresource.go

```diff
@@ -78,6 +79,24 @@ func SpecialResourcesReconcile(ctx context.Context, r *SpecialResourceReconciler
 
 	log = r.Log.WithName(utils.Print(r.parent.Name, utils.Green))
 
+	switch r.parent.Spec.ManagementState {
+	case operatorv1.Force, operatorv1.Managed, "":
+		// The CR must be managed by the operator.
+		// "" is there for completion, as the ManagementState is optional.
+		break
+	case operatorv1.Removed:
+		// The CR associated resources must be removed, even thouth the CR still exists.
+		log.Info("Removed ManagementState, reconciling finalizer")
+		err = removeSpecialResource(ctx, r)
+		return reconcile.Result{}, err
+	case operatorv1.Unmanaged:
+		// The CR must be abandoned by the operator, leaving it in the last known status.
+		log.Info("Unmanaged state, skipping")
```

Comment:
```suggestion
		log.Info("ManagementState=Unmanaged; skipping")
```

Reply:
How about we modify the predicates so that we don't even trigger a reconciliation for unmanaged SpecialResources?

Reply:
This is actually a nice idea, thanks!